src/v2/Components/UserSettings/TwoFactorAuthentication/Components/AppSecondFactor/index.tsx

```diff
@@ -66,6 +80,14 @@ export const AppSecondFactor: React.FC<AppSecondFactorProps> = props => {
     }
   }
 
+  async function onCompleteRedir() {
```

Comment:
Since we're not doing anything with promises / asyncronously / `async / await` no need for `async` here

Reply:
fixed. thanks.